tools/check/ut.go

```
@@ -356,14 +356,13 @@ func main() {
 		fmt.Println("os.Getwd() error", err)
 	}
 
+	var isSucceed bool
 	if len(os.Args) == 1 {
 		// run all tests
-		cmdRun()
```

The error is ignored before this change.